src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGeneratorHelpers.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Interop
+{
+    internal static class ComInterfaceGeneratorHelpers
```

Comment:
Is this file intentionally added?

Reply:
The name is confusing given our complete lack of support fro COM interfaces :)

Reply:
No, this snuck in as part of the cherry-pick. I'll remove it.